.github/workflows/test.yml

```diff
@@ -561,5 +559,185 @@ jobs:
       - uses: codecov/codecov-action@v2
         with:
           files: ./packages/generator-helper/src/__tests__/coverage/clover.xml
-          flags: generator-helper,${{ matrix.os }}
+          flags: generator-helper,${{ matrix.os }},library,binary
+          name: generator-helper-${{ matrix.os }}
+
+  #
+  # Run all tests on macOS and Windows.
+  #
+  # Unlike the other jobs, this job doesn't use Docker (and thus skips some
+  # tests that require dependencies not easily installable without Docker —
+  # specifically, SQL Server), and runs tests on Node.js 12, 14 and 16 (Linux
+  # jobs only run on Node.js 12 and 16 because Node.js 14 is tested in
+  # Buildkite).
```

Comment:
Note to self: update this comment. Node.js 12 and 14 were dropped here as per discussion with @millsp and @Jolg42